openedx/features/offline_mode/tests/test_views.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tests for the testing Offline Mode views.
+"""
+
+from unittest import TestCase
+from unittest.mock import MagicMock, Mock, patch
+
+from opaque_keys.edx.keys import CourseKey
+from openedx.features.offline_mode.views import SudioCoursePublishedEventHandler
+from rest_framework import status
+
+
+class SudioCoursePublishedEventHandlerTestCase(TestCase):
```

Comment:
I added functional tests to this view in my [PR](https://github.com/raccoongang/edx-platform/pull/2576/files#diff-7ad7abf8d10f1532324b3c10fb8521634b7938870e7197bfa9a17024445c5556), so we can actually remove them here.

Reply:
removed